troubadix/standalone_plugins/deprecate_vts.py

```diff
@@ -120,10 +138,34 @@ def _get_summary(content: str) -> Optional[str]:
     return None
 
 
+def find_replacement_oid(
+    file: DeprecatedFile,
+    oid_mapping: Optional[dict[str, str]] = None,
+) -> Optional[str]:
```

Comment:
```suggestion
) -> str | None:
```

I think this is the more _pythonic_ way nowadays and doesnt require a `typing` import